README.md

```diff
@@ -277,62 +287,137 @@ signIn.renderEl(
+    // screen letting them know that an email is on the way.
+    if (res.status === 'ACTIVATION_EMAIL_SENT') {
+      // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
+      return;
+    }
+
+    // The user has completed the registration flow
+    if (res.status === 'REGISTRATION_COMPLETE') {
+      // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - activationToken (string)
       return;
     }
 
     // The user has successfully completed the authentication flow
     if (res.status === 'SUCCESS') {
 
-      // Handle success when the widget is not configured for OIDC
-
       if (res.type === 'SESSION_STEP_UP') {
         // Session step up response
         // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
         // the response will contain user metadata and a stepUp object with the url of the resource
         // and a 'finish' function to navigate to that url
+        // Properties available on `res`:
+        // - status (string)
+        // - user (object)
+        // - type (string)
+        // - stepUp (object)
+        //   - url (string)
+        //   - finish (function)
         console.log(res.user);
         console.log('Target resource url: ' + res.stepUp.url);
         res.stepUp.finish();
         return;
-      } else {
-        // If the widget is not configured for OIDC, the response will contain
-        // user metadata and a sessionToken that can be converted to an Okta
-        // session cookie:
+      }
+
+      if (res.next) {
+        // If the response contains a redirect URL, the response object will contain a
+        // function that redirects the user when called.
+        // Properties available on `res`:
+        // - status (string)
+        // - user (object)
+        // - type (string)
+        // - next (function)
         console.log(res.user);
-        res.session.setCookieAndRedirect('https://acme.com/app');
+        res.next();
         return;
       }
 
+      // If the widget is not configured for OIDC, the default authentication
+      // type of SESSION_SSO is implied. The response will contain user
+      // metadata and a sessionToken that can be converted to an Okta
+      // session cookie:
+      // Properties available on `res`:
+      // - status (string)
+      // - user (object)
+      // - type (string)
+      // - session (object)
+      //   - token (string)
+      //   - setCookieAndRedirect (function)
+      console.log(res.user);
+      res.session.setCookieAndRedirect('https://acme.com/app');
+      return;
+
+    }
+
+  },
+
+  function error(err) {
+    // This function is invoked with errors the widget cannot recover from:
+    // Known errors: CONFIG_ERROR, UNSUPPORTED_BROWSER_ERROR
+  }
+);
+```
+
+**OIDC Configuration:**
```

Comment:
I've divided the example code into two examples, one for non-OIDC configuration and one for OIDC-configurations, so that both examples are comprised of valid code that makes sense, rather than multiple examples in one.

Reply:
It would be great to have a little bit of text explaining why you would use OIDC (you probably should) vs. non-OIDC (probably not, unless your scenario is exotic in some way). Maybe there are some documents we can link to here.

Reply:
@aarongranick-okta I found this on the Dev Docs site, it seems to be a good place to link to: https://developer.okta.com/docs/guides/build-sso-integration/openidconnect/overview/#choosing-a-protocol

Reply:
I can put the OIDC example first and lead with some text nudging the user to use OIDC for new apps and link this document.